makepanda/makepackage.py

```diff
@@ -189,31 +189,65 @@ def MakeInstallerNSIS(version, file, title, installdir, compressor="lzma", **kwa
     oscmd(cmd)
 
 
-def MakeDebugSymbolArchive(zipname, dirname):
-    outputdir = GetOutputDir()
-
+def MakeDebugSymbolZipArchive(zipname):
     import zipfile
 
-    zip = zipfile.ZipFile(zipname, 'w', zipfile.ZIP_DEFLATED)
+    outputdir = GetOutputDir()
+    zip = zipfile.ZipFile(zipname + '.zip', 'w', zipfile.ZIP_DEFLATED)
 
     for fn in glob.glob(os.path.join(outputdir, 'bin', '*.pdb')):
-        zip.write(fn, dirname + '/bin/' + os.path.basename(fn))
+        zip.write(fn, 'bin/' + os.path.basename(fn))
```

Comment:
Does this remove the top-level directory from the archives?  Why?

Reply:
To stay consistent with the archive that 7-zip creates. 7-zip has no feature that I know of that allows the top-level directory name to be set, so it doesn't create one. It is possible to make 7-zip emit an archive with a top-level directory, BUT it will be called either `built_x64` or `built`, it does not match the original name unfortunately.

`dirname` in this case was `C:\Panda3D-1.11.0-x64`, but the zipfile that was created had a top level directory of `Panda3D-1.11.0-x64` anyway which is wrong behavior on its own.

Reply:
If we want to keep the top-level directory, we can use 7-Zip's `rn` option to rename the top level directory after creating the original archive, but it does require two invocations of the 7-Zip command line utility. Should the top-level directory be restored?